app/i18n.js

```diff
@@ -7,24 +7,18 @@
  *   script `extract-intl`, and must use CommonJS module syntax
  *   You CANNOT use import/export in this file.
  */
-const addLocaleData = require('react-intl').addLocaleData; //eslint-disable-line
+
+const { addLocaleData } = require('react-intl');
```

Comment:
[`addLocaleData` is removed in latest `react-intl`.](https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#breaking-api-changes)